From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Wolfgang Bumiller <w.bumiller@proxmox.com>
Date: Tue, 10 Mar 2020 12:55:22 +0100
Subject: [PATCH 25/32] PVE: monitor: disable oob capability

A bisect revealed that commit 8258292e18c3
("monitor: Remove "x-oob", offer capability "oob" unconditionally")
causes unexpected hangs when restoring live snapshots from some
types of block devices (particularly RBD).
We need to figure out what's happnening there. For now, since we
had this disabled before and probably don't need it now either,
disable oob, so we can get a functioning qemu out...

Signed-off-by: Wolfgang Bumiller <w.bumiller@proxmox.com>
Signed-off-by: Thomas Lamprecht <t.lamprecht@proxmox.com>
---
 monitor/qmp.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/monitor/qmp.c b/monitor/qmp.c
index b67a8e7d1f..8f44fed944 100644
--- a/monitor/qmp.c
+++ b/monitor/qmp.c
@@ -395,8 +395,7 @@ void monitor_init_qmp(Chardev *chr, bool pretty)
     MonitorQMP *mon = g_new0(MonitorQMP, 1);
 
     /* Note: we run QMP monitor in I/O thread when @chr supports that */
-    monitor_data_init(&mon->common, true, false,
-                      qemu_chr_has_feature(chr, QEMU_CHAR_FEATURE_GCONTEXT));
+    monitor_data_init(&mon->common, true, false, false);
 
     mon->pretty = pretty;
 
